odl/tomo/geometry/spect_geometry.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2014 - 2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""SPECT geometry."""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+from builtins import super
+
+# External
+import numpy as np
+
+# Internal
+import odl
+from odl.tomo.geometry.detector import Flat2dDetector
+from odl.tomo.geometry.geometry import (Geometry,
+                                        AxisOrientedGeometry)
+from odl.tomo.util.utility import perpendicular_vector
+
+__all__ = ('ParallelHoleCollimatorGeometry', )
+
+
+class ParallelHoleCollimatorGeometry(Geometry, AxisOrientedGeometry):
```

Comment:
I'm getting more and more convinced that it is actually the **Detector** which is special in this kind of SPECT geometry, and that it would make sense to create a `Flat2dCollimatedDetector` subclass of `Flat2dDetector`. That new guy would then allow to define the field of view of an individual detector pixel (I think it is safe to assume that it's the same for all pixels).